app/src/main/java/io/intrepid/contest/screens/contestoverview/DualCategoryScoreAdapter.java

```diff
@@ -0,0 +1,69 @@
+package io.intrepid.contest.screens.contestoverview;
+
+import android.support.v7.widget.RecyclerView;
+import android.view.ViewGroup;
+
+import java.util.List;
+
+import io.intrepid.contest.models.Category;
+import io.intrepid.contest.models.ScoreWeight;
+import io.intrepid.contest.screens.contestcreation.categorieslist.CategoryViewHolder;
+
+class DualCategoryScoreAdapter extends RecyclerView.Adapter {
+    private final int CATEGORY_VIEW_TYPE = 111;
```

Comment:
These constants can be anything you define it to be since i don't believe Android has any predefined viewtype constants for viewholders